src/subscribe.rs

```diff
@@ -0,0 +1,117 @@
+struct Subscriber {
+    name: Option<String>,
+    email: String,
+    phone: Option<String>,
+}
+
+impl Subscriber {
+    fn to_provider_data(&self) -> ft_sdk::auth::ProviderData {
+        ft_sdk::auth::ProviderData {
```

Comment:
Subcription is not auth provider, it should not use ft_sdk::auth::ProviderData. The email stuff should be added a provider data, but for subscription (and any other app that wants to store user preferences etc) we should have another method.

Reply:
> If there is no user in the `fastn_user` table, this route creates one.
(from README.md:26)

you have to call `ft_sdk::auth::provider::create_user` with `ProviderData` to do this. FWIW, I am using "email" as the provider id to for this.